### DISCUSSION TITLE
VoxelData

### DISCUSSION THREAD

**Discussion by Gorfonous** [Eldemarkki/Marching-Cubes-Terrain#57] [Q&A]
in minecraft there are differnt types of blocks like dirt stone and ore, does this support something like that? if not is there a plan to implement it? It would be cool if you could say "only remove dirt" or "only remove stone/ore" and know how much is being removed.

**Comment by Eldemarkki** [comment]
No, currently not. There is however a way to paint the terrain with different colors, but I don't think that is what you are looking for. I haven't thought about adding different block types, but I may consider it in the future.
